Update config_rb_server.rst (#2085)
chef/chef-web-docs: chef_master/source/config_rb_server.rst

      550 = 350 + [(4 - 2) * (25 * 4)]

.. end_tag

``postgresql['sslmode']``

>    SSL encryption mode between Chef Server and PostgreSQL.  Valid settings are ``'disable'`` (off) and ``'require'`` (on).  Default value: ``'disable'``. 

Reviewer: ```suggestion
   SSL encryption mode between the Chef Infra Server and PostgreSQL.  Valid settings are ``'disable'`` and ``'require'``.  Default value: ``'disable'``. 
```
